fe/global.css

```diff
@@ -60,55 +57,77 @@
 }
 
 :global a {
-  text-decoration: unset !important;
-  color: unset !important;
+  text-decoration: none !important;
+  color: #5e73e5;
 }
 
 :global body h1 {
   color: colorsTextMain;
-  font-family: "Montserrat";
-  font-weight: 600;
+  font-family: "Poppins";
+  font-weight: 500;
   letter-spacing: -0.03em;
-  line-height: 60px !important;
-  font-size: 45px !important;
+  line-height: 58px !important;
+  font-size: 31px !important;
   margin: 0px !important;
+  color: #2f2d44;
 }
 
 :global body h2 {
   color: colorsTextMain;
-  font-family: "Montserrat";
-  font-weight: 600;
+  font-family: "Poppins";
+  font-weight: 500;
   letter-spacing: -0.02em;
-  line-height: 54px;
-  font-size: 40px;
-  margin: 0px;
+  line-height: 22px !important;
+  font-size: 22px !important;
+  color: #2f2d44;
+  margin-top: 60px !important;
+  padding-top: 60px !important;
```

Comment:
In general a lot of `!important` tags is a red flag when it comes to css. 

Looking in the browser, it looks like we don't need these `!important`'s, as this css is just overriding the h2 on blueprint anyway (as it's more specific).

I would prefer we got rid of these.

Reply:
Also with blueprint 3.0, the typography styles aren't automatically applied (you have to opt in on a heading) so this definitely won't be a problem when we get around to integrating 3.0 into the docs site.